--- src/money.3.c	2013-04-18 03:19:01.000000000 -0700
+++ src/money.4.c	2013-04-18 03:19:01.000000000 -0700
@@ -1,26 +1,31 @@
 #include <stdlib.h>
 #include "money.h"
 
+struct Money
+{
+  int amount;
+};
+
 Money *
 money_create (int amount, char *currency)
 {
   return NULL;
 }
 
 int
 money_amount (Money * m)
 {
-  return 0;
+  return m->amount;
 }
 
 char *
 money_currency (Money * m)
 {
   return NULL;
 }
 
 void
 money_free (Money * m)
 {
   return;
 }
